pkg/controllers/dashboard/fleetcharts/controller_test.go

```diff
@@ -147,6 +142,12 @@ func Test_ChartInstallation(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			ctrl := gomock.NewController(t)
+			configCache := ctrlfake.NewMockCacheInterface[*v1.ConfigMap](ctrl)
```

Comment:
I don't think we need to make a new controller for each test. This could be moved above to the set up portion.

Reply:
We need a new gomock.Controller per `*testing.T`